test/command_monitoring/pre-42-server-connection-id.json

```diff
@@ -0,0 +1,101 @@
+{
+  "description": "pre-42-server-connection-id",
```

Comment:
We have not implemented the server connection id spec yet so let's remove these test files.